cv_next/app/feed/page.tsx

```diff
@@ -1,16 +1,18 @@
-import CVItem from "@/components/CVItem"
-import CVModel from "@/types/cv"
+import { Suspense } from "react";
+import Feed from "./components/feed";
+import { handleCurrentUser } from "../actions/users/handleUserState";
+import { redirect } from "next/navigation";
 
-export default function Feed() {
-  let mockcv = new CVModel(
-    "cv-id",
-    "user-id",
-    "https://lh3.google.com/u/0/d/1gquDwmvOjuRCVQJ-LkMnA3NOLWDw_UDu=w1000-h1200-p",
-    1
-  )
+export default async function Page() {
+  const redirectOutput = await handleCurrentUser("/feed");
```

Comment:
BTW you'd probably want this in every Page. I believe you can HOC all pages and add this functionality in a more convenient way.

Reply:
Now that we managed to solve the problem with the SSR and the login in https://github.com/magshimim-next/cv-next/issues/72 we can actually remove this as the middleware handles this
Just want to wait for the merge of the ssr fix to fully delete